WordPress/src/main/res/values/strings.xml

```diff
@@ -1362,6 +1362,8 @@
     <string name="reader_btn_unfollow">Following</string>
     <string name="reader_add_bookmark">Add to saved posts</string>
     <string name="reader_remove_bookmark">Remove from saved posts</string>
+    <string name="reader_btn_bookmark">SAVE</string>
```

Comment:
Wouldn't be better to use the `textAllCaps` attribute instead of having strings written in capitals? WDYT?